docs/tutorials/using-ps3-sixaxis.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Using the EV3 Buttons
+subject: Hardware - PS3 Sixaxis bluetooth gamepad
+author: Anton Vanhoucke
+---
+
+* Table of Contents
+{:toc}
+
+# What you need
+A PS3 gamepad (also known as Sixaxis controller or Dualshock 3)
+A mini-usb / usb cable
+A working ssh and internet connection to the Ev3 (or other ev3dev device).
+
+# Connection
+The PS3 controller pairs more or less like a normal bluetooth device in brickman. First make sure bluetooth is on in Ev3dev and the brick is visible. Next you have to connect the gamepad via a mini usb cable to the Ev3. Next press the PS3 button on the gamepad. The controller should now show up in brickman under wireless > bluetooth. Connect, pair and remove the USB cable. Whenever you press the PS3 button on the gamepad now, it will try to connect to the ev3 brick. Nice!
+
+If brickman doesn't work or if you don't have a display, like on a BrickPi, bluetoothectl is the way to go. A nice tutorial is here: https://wiki.gentoo.org/wiki/Sony_DualShock
+
+
+# Installing the necessary python libraries
+To read gamepad values into our python program we need to install the evdev library. (This is not a typo. There is no 3 in evdev)
+
+Install it like so:
```

Comment:
we package `python-evdev`, so no need to use `pip`. Just `sudo apt-get update; sudo apt-get install python-ev3dev` (or `python3-evdev`).